src/components/v5/shared/HamburgerButton/HamburgerButton.tsx

```diff
@@ -0,0 +1,67 @@
+import clsx from 'clsx';
+import React, { FC } from 'react';
+import { HamburgerButtonProps } from './types';
+
```

Comment:
Can you check if there is already a similar component defined?

Reply:
Same as previously - I need a different logic here